x-pack/plugins/apm/public/components/app/correlations/latency_correlations.tsx

```diff
@@ -293,9 +242,14 @@ export function LatencyCorrelations({ onFilter }: { onFilter: () => void }) {
           field: sortField,
           direction: sortDirection,
         },
-      } as EuiTableSortingType<MlCorrelationsTerms>,
+      } as EuiTableSortingType<LatencyCorrelation>,
```

Comment:
nit: afaict there's no need to memoize `sorting`.

```ts
  const sorting = {
    sort: { field: sortField, direction: sortDirection },
  } as EuiTableSortingType<LatencyCorrelation>;

  const histogramTerms = useMemo(() => {
    return orderBy(
      response.latencyCorrelations ?? [],
      sortField,
      sortDirection
    );
  }, [response.latencyCorrelations, sortField, sortDirection]);
```

Reply:
Updated in 6076fbf.